backend/models/cans.py

```diff
@@ -711,3 +757,15 @@ class CANFiscalYearFundingDetails(BaseModel):
     can_fiscal_year_id: Mapped[Optional[int]] = mapped_column(
         Integer, ForeignKey("can_fiscal_year.id")
     )
+
+class CANAppropriationDetails(BaseModel):
+    """
+
+    """
+
+    __tablename__ = "can_appropriation_details"
+
+    id: Mapped[int] = BaseModel.get_pk_column()
+    appropriation_prefix: Mapped[Optional[str]] = mapped_column(String)
```

Comment:
The `= mapped_column(String)` is redundant but doesn't hurt anything if you keep it in.